css/main.css

```diff
@@ -1,3 +1,58 @@
 /* Add your custom CSS here */
+*{
+    margin: 0px;
+}
+body {
+    font-family: "https://fonts.google.com/specimen/Quicksand";
+    color: rgb(0,0,0); 
+}
+
+.img-prof {
+    border-radius: 40%;
+}
+a {
+    padding:5px;
+    ;
+}
+
+.flex-nav{
+    display: flex;
+    justify-content: space-between;
+    padding: 5vh;
+}
+h1 h2 h3 h4 h5 h6 {
+    font-family: "https://fonts.google.com/specimen/Poppins";
+}
+.background-beige{
+    background-color: #E6DACD;
+    margin: 0px;
+    height: 50vh;
+}
+.container-class{
+    margin: 0px;
+}
```

Comment:
Again, don't set your own margins on a class that is applied to a Bootstrap element (this is used on a `container` element)